openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -131,7 +131,7 @@ class AccountViewSet(ViewSet):
 
             GET /api/user/v1/me[?view=shared]
             GET /api/user/v1/accounts?usernames={username1,username2}[?view=shared]
-            GET /api/user/v1/accounts?email={user_email}
+            GET /api/user/v1/accounts?email={user_email1,user_email2}
```

Comment:
Shouldn't this be `GET /api/user/v1/accounts?email={user_email1},{user_email2}`

Reply:
Can you share an actual request with some dummy data?

Reply:
GET http://localhost:18000/api/user/v1/accounts?email=edx@example.com,staff@example.com

Response:
```
[
    {
        "account_privacy": "private",
        "profile_image": {
            "has_image": false,
            "image_url_full": "http://localhost:18000/static/images/profiles/default_500.png",
            "image_url_large": "http://localhost:18000/static/images/profiles/default_120.png",
            "image_url_medium": "http://localhost:18000/static/images/profiles/default_50.png",
            "image_url_small": "http://localhost:18000/static/images/profiles/default_30.png"
        },
        "username": "edx",
        "email": "edx@example.com",
        "id": 3,
    },
    {
        "account_privacy": "private",
        "profile_image": {
            "has_image": false,
            "image_url_full": "http://localhost:18000/static/images/profiles/default_500.png",
            "image_url_large": "http://localhost:18000/static/images/profiles/default_120.png",
            "image_url_medium": "http://localhost:18000/static/images/profiles/default_50.png",
            "image_url_small": "http://localhost:18000/static/images/profiles/default_30.png"
        },
        "username": "staff",
        "email": "staff@example.com",
        "id": 8,
    }
]
```